doc/content/tutorials/KRUSTY.md

```diff
@@ -0,0 +1,885 @@
+
+!listing /tutorials/krusty/KRUSTY/solide.i
+  start=Postprocessors
+
+### Neutronics Input Files
+
+We begin by defining a mesh on which OpenMC will receive temperature from the couple MOOSE application.
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  end=AuxVariables
+
+Then, we can define our initial conditions for the temperature, which will be a constant function of $1073 K$
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  start=ICs
+  end=Problem
+
+Next, we define the `[Problem]` block, which will describe all objects necessary for the physics solve. In order to replace MOOSE finite element calculations with OpenMC particle transport calculations, we are going to use the [OpenMCCellAverageProblem](https://cardinal.cels.anl.gov/source/problems/OpenMCCellAverageProblem.html) class. In this model, we specify the total fission power by which to normalize OpenMC's tally results. Next, we indicate which blocks in the `[Mesh]` should be considered for temperature feedback using the `temperature_blocks` parameter. Mesh tallies were used to estimate the heat source, as indicated by `tally_type`, where the same mesh can be translated to multiple locations in the OpenMC geometry, specifying a mesh file using `mesh_template`.
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  block=Problem
+  
+Next, we add a series of auxiliary variables for solution visualization 
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  block=AuxVariables
+
+In this example, the overall calculation workflow is as follows:
+
+1. Run OpenMC using the initial condition for the temperature.
+2. Run MOOSE heat conduction with a given power distribuition from OpenMC.
+3. Run OpenMC with an updated temperature distribution.
+4. Extract the kappa-fission distribution (the recoverable fission energy)
+  computed by OpenMC and map in the opposite direction from OpenMC cells
+  to all the MOOSE elements that mapped to each cell.
+
+The above sequence is repeated until desired convergence of the coupled domain is achieved.
+The [MultiApps](https://mooseframework.inl.gov/syntax/MultiApps/index.html)
+and [Transfers](https://mooseframework.inl.gov/syntax/Transfers/index.html)
+blocks describe the interaction between Cardinal and MOOSE. OpenMC is run as the main application, with
+MOOSE heat conduction run as the sub-application.
+
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  start=MultiApps
+  end=Postprocessors
+
+For the heat source transfer from OpenMC, we ensure conservation by requiring that the integral of heat source computed by OpenMC matches the integral of the heat source received by MOOSE. We also add a postprocessor to evaluate the Tally relative error and the multiplication factor.
+
+!listing /tutorials/krusty/KRUSTY/openmc.i
+  block=Postprocessors
+
+Because we did specify sub-cycling in the `[MultiApps]` block, we do not need to define a number of time steps here.
```

Comment:
This isn't necessarily because we used sub-cycling, but rather because we are okay with OpenMC using the default `dt` of 1.0 (so we don't need to specify it ourselves).